android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -882,6 +882,23 @@
 		"obsoleteTech": "Combined Arms",
 		"attackSound": "horse"
 	},
+	{
+		"name": "Hakkapeliitta",
+		"unitType": "Mounted",
+		"replaces": "Lancer",
+		"uniqueTo": "Sweden",
+		"movement": 4,
+		"strength": 25,
+		"cost": 185,
+		"requiredTech": "Metallurgy",
+		"requiredResource": "Horses",
+		"uniques": ["Can move after attacking","No defensive terrain bonus","-[33]% Strength vs [City]",
+			"Transfer Movement to General", "+[15]% Combat Bonus when stacked with Great General"],
```

Comment:
As far as I understand it, we're trying to move away from `combat bonus` and towards `strength`. Also, you can parametrize the unit names here, so imo it's better to use `[+15]% Strength when stacked with [unitFilter] units` and `Transfer Movement to [unitFilter] units`

Reply:
I changed the + Strength when stacked part, now it makes less assumptions.
I _could_ parametrize the transfer movement one as well, but the problem is that it is checked during the great generals start of turn. So if it was made generic every unit in the game would have to check every turn if they are stacked with a unit that can give them extra movement, which seems unneccesary?

Reply:
Good point, I'll leave that for @yairm210 to decide.

Reply:
I think that's a fair check. Most units are the only unit on their tile anyway, and "once per turn" isn't a heavy cost.